metabase/metabase
# Refactor - Break ObjectDetail down

### Description

It was hard to follow `ObjectDetail.tsx`. It was over 600 LOC and contained multiple component definitions. 
I already wanted to break it down when working on #32320.
Now I'm working on #32720 and the need to break it down emerged again.

This PR only moves code around and updates imports (changes long absolute import paths to relative imports where possible, groups and sorts according to our eslint config).

This PR is going to be backported to help avoid conflicts.

### How to verify

There should be no difference in how Metabase works.
Opening object detail modal for a model object should work (create and save a model, visualize it as a table, and then click on an id of object in the table).
